action/README.md

```diff
@@ -0,0 +1,58 @@
+# deployctl GitHub Action
+
+The Deno Deploy GitHub Action allows for easy deployment to Deno Deploy from a
+GitHub Actions workflow. This allows a simple way to have a build step before
+deploying to Deno Deploy.
+
+> ⚠ If your project does not require a build step, we recommend you use the
+> ["Automatic" deployment mode][automatic-mode] of our GitHub integration. It is
+> faster and requires no setup.
+
+## Usage
+
+To deploy you just need to include the Deno Deploy GitHub Action as a step in
+your workflow.
+
+You do **not** need to set up any secrets for this to work. You **do** need to
+link your GitHub repository to your Deno Deploy project and choose the "GitHub
+Actions" deployment mode. You can do this in your project settings on
+https://dash.deno.com.
+
+```yml
+- name: Deploy to Deno Deploy
```

Comment:
We should add a note about `permissions` for the step? Or include it in the setup yaml.